src/Language/Mulang/Analyzer/EdlQueryCompiler.hs

```diff
@@ -153,7 +153,7 @@ compileInspection = f
   f "UsesComposition"                  E.Unmatching   = plain usesComposition
   f "UsesComprehension"                E.Unmatching   = f "UsesForComprehension" E.Unmatching
   f "UsesConditional"                  E.Unmatching   = plain usesConditional
-  f "UsesDyamicPolymorphism"           E.Unmatching   = contextual usesDyamicPolymorphism'
+  f "UsesDynamicPolymorphism"           E.Unmatching   = contextual usesDynamicPolymorphism'
```

Comment:
ever so slightly bothered by the indentation here :laughing:

Reply:
:rofl: